plugins/plugin-webpack/plugin.js

```diff
@@ -97,6 +97,7 @@ function emitHTMLFiles({doms, jsEntries, stats, baseUrl, buildDirectory, htmlMin
 }
 
 function getSplitChunksConfig({numEntries}) {
+  const isCssModule = module => module.type === `css/mini-extract`
```

Comment:
"CSS Module" is a bit of an overloaded term that means something more specific in Snowpack. Would prefer `isCss` or` isModuleTypeCss`